instance/factories.py

```diff
@@ -147,6 +147,9 @@ def production_instance_factory(**kwargs):
         "demo_test_users": [],
         # Disable certificates process to reduce load on RabbitMQ and MySQL
         "SANDBOX_ENABLE_CERTIFICATES": False,
+        "EDXAPP_EXTRA_REQUIREMENTS": [
+            { "name": "git+https://github.com/mitodl/edx-sysadmin.git@v0.1.3#egg=edx-sysadmin==v0.1.3" },
```

Comment:
Can you add an env variable for the version, something like `EDXAPP_EDX_SYSADMIN_VERSION`? Then we can update it when we upgrade to maple etc., without having to update the code.

Reply:
@pomegranited do you mean a variable for ocim, or a variable to go in the ansible vars?

Reply:
Hmm.. I was thinking a variable for Ocim, but maybe it would be better to move this whole block into config, and read it into the Ocim settings?

```yaml
PRODUCTION_INSTANCE_EXTRA_CONFIGURATION:
  # Don't create default users on production instances
  DEMO_CREATE_STAFF_USER: false
  demo_test_users: [],
  # Disable certificates process to reduce load on RabbitMQ and MySQL
  SANDBOX_ENABLE_CERTIFICATES: false
  EDXAPP_EXTRA_REQUIREMENTS
  - name: git+https://github.com/mitodl/edx-sysadmin.git@v0.1.3#egg=edx-sysadmin==v0.1.3 
```

So you can just do this:
```
production_settings = settings.PRODUCTION_INSTANCE_EXTRA_CONFIGURATION
```

That way, if/when we have other apps that we need to install on production instances, we can just add them to the configuration, including any extra env variables or whatever settings they need.

Reply:
Nice :ok_hand: